File: src/net/eduard/api/command/api/ApiHelpCommand.java

package net.eduard.api.command.api;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;

public class ApiHelpCommand extends CommandManager {

	public ApiHelpCommand() {
		super("help","ajuda","?");
		setDescription("Mostra uma lista de comandos");
	
		
	}
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
		for (CommandManager sub : getCommand("api").getCommands().values()) {
			if (sender.hasPermission(sub.getPermission())) {
				sender.sendMessage("§a"+ sub.getUsage() + " §8-§7 " + sub.getDescription());
			}
		}
		
		return true;
	}
	

}

File: src/main/java/net/eduard/api/command/api/ApiHelpCommand.kt
package net.eduard.api.command.api

import net.eduard.api.lib.manager.CommandManager
import org.bukkit.command.Command
import org.bukkit.command.CommandSender

class ApiHelpCommand : CommandManager("help", "ajuda", "?") {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        for (sub in getCommand("api").subCommands.values) {
            if (sender.hasPermission(sub.permission)) {
                sender.sendMessage("§a" + sub.usage + " §8-§7 " + sub.description)
            }
        }
        return true
    }

    init {
        description = "Mostra uma lista de comandos"
    }
}